enlighten/common.py

```diff
@@ -13,7 +13,7 @@
       can be modules (files) within it
 """
 
-VERSION = "4.0.26"
```

Comment:
I'd rather not jump to 4.1.0 with this commit.  The first 4.1.0 release should be for Windows and on our website, not a Mac RC.

Reply:
Hi Samie, I think I'd rather go with my Mac branch instead of this one. I don't want to start committing the generated Python files.

Reply:
Sounds good. I'll close this one